libs/core/hardware/CMakeLists.txt

```diff
@@ -26,6 +27,7 @@ set(hardware_compat_headers
     hpx/util/hardware/cpuid/linux_x86.hpp => hpx/modules/hardware.hpp
     hpx/util/hardware/cpuid/msvc.hpp => hpx/modules/hardware.hpp
     hpx/util/hardware/timestamp/bgq.hpp => hpx/modules/hardware.hpp
+    hpx/util/hardware/timestamp/cuda.hpp => hpx/modules/hardware.hpp
```

Comment:
This is not necessary (the file is new so there's no need for a compatibility header).

Reply:
@NK-Nikunj Would you be able to fix this before the release?

Reply:
yes, I'm currently working on it. I believe we can get it into the release.